.travis.yml

```diff
@@ -24,6 +24,7 @@ script:
   - ls -la bin/linux/amd64
   - amp --server=localhost cluster create --tag=local --registration=none --notifications=false
   - TESTINCLUDE=platform/testing testrunner platform/tests
+  - tests/run
```

Comment:
Ah, I see ... I didn't look at it closely enough, but I understand the problem you're trying to solve: you want tests that can run inside the swarm. In that case, we can look at optimizing the build time by using an existing image that already has all the tools needed (like `amptools`) (or we can create a lighter weight one specifically for these tests). This is an optimization we can worry about later. Thanks!